doc/content/integrations/node-red/send/_index.md

```diff
@@ -4,32 +4,50 @@ description: ""
 weight: 3
 ---
 
-This section follows the process of setting up a flow which publishes messages to a certain topic that the MQTT Server is subscribed to.
+This section explains the process of setting up a flow which publishes messages to a certain topic that the MQTT Server is subscribed to.
 
 Doing this schedules downlink messages to be sent to your end device. This section follows the example for publishing downlink traffic in [MQTT Server]({{< ref "/integrations/mqtt" >}}) guide.
 
 ## Configure MQTT Out Node
 
-Find the **mqtt out** node in the Node-RED library and place it on the dashboard. This node connects to the MQTT broker and publishes messages. Configure the **Server** options with the same settings as used in the [Receive Events and Messages]({{< ref "/integrations/node-red#receive-events-and-messages" >}}) section before.
+1. Place the **mqtt out** node on the dashboard. 
 
-As stated in [MQTT Server]({{< ref "/integrations/mqtt" >}}) guide, topic `v3/{application_id}/devices/{device_id}/down/push` can be used for scheduling downlink messages. Choose a **QoS** from listed options and state whether you want the MQTT Server to retain messages. 
+2. Configure the **Server** options with the same settings as in the [Receive Events and Messages]({{< ref "/integrations/node-red#receive-events-and-messages" >}}) section.
+
+3. Set **Topic** to `v3/{application_id}/devices/{device_id}/down/push` to schedule downlink messages (as stated in [MQTT Server]({{< ref "/integrations/mqtt" >}}) guide). 
+
+4. Choose a **QoS** from listed options and state whether you want the MQTT Server to retain messages. 
 
 {{< figure src="mqtt_out_node_properties.png" alt="mqtt out node properties" >}}
 
-Further, find and place the **inject** node on the dashboard. This node injects a message into a flow and can be triggered manually, but it is also possible to define regular intervals between the automatic injections. Double-click on the **inject** node to configure its properties. The **Payload** field can have different formats - in this example, JSON is used. Expand the **Payload** field and enter this:
+## Configure Inject Node
+
+1. Place the **inject** node on the dashboard. Double-click on the node to configure its properties. 
+
+2. Choose **buffer** under **Payload** and enter the payload you wish to send. 
```

Comment:
So this is in array form, right, so `[0, 1, 2, 4]` for bytes `01020304` (as hex)